x-pack/solutions/security/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -43,7 +43,7 @@ export interface ProcessDeps {
   process: Process;
   isSessionLeader?: boolean;
   depth?: number;
-  onProcessSelected?: (process: Process) => void;
+  onProcessSelected?: (process: Process, userAction?: boolean) => void;
```

Comment:
`userActions` somewhat implies you are passing an action prop.. have you considered calling it `showPanel` or `isPanelOpen`?

Reply:
hmmm yeah I can see how `userAction` could be a bit confusing. I wanted to use something that would give on indication on the source of the event. Naming them `showPanel` could be confusing because that property is basically only used when the sessionview is in the flyout...
I'll think about other options, maybe the @elastic/cloud-security-posture team will have some ideas also

Reply:
maybe something like `userSelected` or `userSelection`... Doesn't seem great either though, I'll keep thinking!

Reply:
what about `isManualSelection`?

Reply:
`isManualAction` it is! :)